### PR TITLE
Update Russian translations

### PR DESCRIPTION
**Pull request by EleoXDA**

### CODE REVIEW

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -12,17 +12,39 @@
...
+        <item quantity="one">Держите %d день</item>
+        <item quantity="other">Держите %d дня</item>
+    </plurals>
+
+    <!-- Output format bottom sheet -->
+    <string name="output_format_bottom_sheet_output_format">Выходной формат</string>
+    <string name="output_format_bottom_sheet_compression_level">Уровень сжатия</string>
+    <string name="output_format_bottom_sheet_bitrate">Битрейт</string>
+    <string name="output_format_bottom_sheet_sample_rate">Частота дискретизации</string>
+    <string name="bottom_sheet_reset">Reset to defaults</string>
```
Comment (chenxiaolong) [inline app/src/main/res/values-ru/strings.xml:39]: Did you mean to add a translation for `Reset to defaults`?

Comment (EleoXDA) [reply app/src/main/res/values-ru/strings.xml:39]: Oops I think I missed that. Let me check again

**Review by chenxiaolong — APPROVED** [review]

**Comment by PatrykMis** [comment]
Update of Russian translation is needed again - plurals.

Lint throws the following error:

> For locale "ru" (Russian) the following quantities should also be defined: few, many

I don't know Russian but "other" may be equivalent to "zero".

**Comment by EleoXDA** [comment]
> Update of Russian translation is needed again - plurals.
> 
> Lint throws the following error:
> 
> > For locale "ru" (Russian) the following quantities should also be defined: few, many

That is correct. In "ru" , 1 is different, 2-4 is different and 5-infinity is different.

**Comment by EleoXDA** [comment]
Added a PR to fix:
https://github.com/chenxiaolong/BCR/pull/117

**Pull request merged**